src/styles/reset.scss

```diff
@@ -0,0 +1,12 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
```

Comment:
I am not sure that this properties must be in reset file

Reply:
Agree, it's not critical, but looks strange